ngx-fudis/projects/ngx-fudis/src/lib/components/notification/notification.component.scss

```diff
@@ -0,0 +1,57 @@
+@use 'sass:map';
+@use '../../foundations/colors/mixins.scss' as colors;
+@use '../../foundations/focus/mixins.scss' as focus;
+@use '../../foundations/borders/mixins.scss' as borders;
+@use '../../foundations/typography/mixins.scss' as typography;
+@use '../../foundations/spacing/tokens.scss' as spacing;
+
+.fudis-notification {
+	@include borders.fudis-border-radius('2px');
+
+	display: flex;
+	align-items: flex-start;
+	margin: 0 spacing.$fudis-spacing-xxs;
+	padding: 0 spacing.$fudis-spacing-s 0 spacing.$fudis-spacing-xxs;
+	width: 100%;
+	min-height: spacing.$fudis-spacing-m;
+
+	&__content {
+		padding-top: spacing.$fudis-spacing-xs;
```

Comment:
Onko tämä haluttu jättää näin, eikä lähdetä kikkailemaan spacing +- spacing jutuilla. Eli tämä on paras keskitys mitä meidän tokeneilla saa aikaan?

<img width="642" alt="Screenshot 2023-03-01 at 12 02 06" src="https://user-images.githubusercontent.com/92301174/222107123-58c86a93-d3cd-467d-bc6e-3b6b0abf8ce9.png">